Assets/Scripts/SceneManagement/SceneManifest.cs

```diff
@@ -29,7 +31,9 @@ public static string Translate(SceneManifest scene)
         sceneNames.Add(SceneManifest.TestLevel, "testLevel");
         sceneNames.Add(SceneManifest.Title, "Title");
         sceneNames.Add(SceneManifest.Credits, "Credits");
-        sceneNames.Add(SceneManifest.DummyLevel, "DummyLevel");
+        sceneNames.Add(SceneManifest.DummyScene0, "DummyScene0");
+        sceneNames.Add(SceneManifest.DummyScene1, "DummyScene1");
+        sceneNames.Add(SceneManifest.SceneChanger, "SceneChanger");
```

Comment:
This map should be static, and all of the adds to the map should be in a static code section. (See https://stackoverflow.com/questions/8459095/static-code-blocks). Basic principle is that if you *can* only do something once, only to it once.